website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6416,6 +6421,68 @@ with pipeline as p:
 
 4. After the job has been submitted to the Beam runner, shutdown the expansion service by terminating the expansion service process.
 
+#### 13.2.3. Using cross-language transforms in a Go pipeline
+
+If a Go-specific wrapper for a cross-language is available, use that; otherwise, you have to use the
+lower-level [CrossLanguage](https://pkg.go.dev/github.com/apache/beam/sdks/go/pkg/beam#CrossLanguage)
+function to access the transform.
+
+**Expansion Services**
+
+The Go SDK does not yet support automatically starting an expansion service. In order to use
+cross-language transforms, you must manually start any necessary expansion services on your local
+machine and ensure they are accessible to your code during pipeline construction.
```

Comment:
I'm on the fence about adding a JIRA for folks to reference to contribute this feature. As you like. Probably better after we add the Xlang override/registration if needed.

Reply:
Why not? I added one anyway and just left a note saying that it may be easier to implement after Xlang override/registration is added.